src/components/WorkChart.astro

```diff
@@ -0,0 +1,17 @@
+---
+type Props = {
+  img: string;
+  chartModifier: string | null;
+  href: string | null;
+};
+
+const { img, chartModifier,href} = Astro.props;
+---
+
+<div class="work-chart">
+    <img src={img} alt="" class="work-chart__image">
+    <div class="work-chart__content">
+        <div class={`work-chart__content-text ${chartModifier}`}> <slot /> </div>
```

Comment:
If this is just gonna be a heading, you could use a Heading tag like h3, h4 to make it more semantic instead of just a div tag.

Reply:
In fact since you are sending a <p>text</p> You could just change the div to p tag and send just the pure text to the component.